lib/Rex/Virtualization.pm

```diff
@@ -47,6 +47,11 @@ sub create {
     $wanted_provider = $wanted_provider->{type} || "LibVirt";
   }
 
+  if ( !$wanted_provider ) {
+    die
+      "No virtualization provider set.\nPlease use <set virtualization => 'YourProvider';> to set one.\n";
```

Comment:
Would it be a good idea to:
- use backticks instead of `<>` for marking code/commands
- also point the user to docs
?

I'm thinkink about something like this:
```
"No virtualization provider set.\nPlease use `set virtualization => 'YourProvider';` to set one,\nor see `perldoc Rex::Commands::Virtualization` for more options";
```